ui/src/pages/Configuration/ConfigurationPage.tsx

```diff
@@ -38,24 +40,26 @@ const Row = styled(ColumnLayout.Row)`
     }
 `;
 
+type TabSchema = z.infer<typeof TabSchema>;
+
 function ConfigurationPage() {
     const unifiedConfigs = getUnifiedConfigs();
     const { title, description, subDescription, tabs } = unifiedConfigs.pages.configuration;
     const permittedTabNames = useMemo(() => tabs.map((tab) => tab.name), [tabs]);
-    const isComponentMounted = useRef(false);
+    const isComponentMounted = useRef<boolean>(false);
 
-    const [activeTabId, setActiveTabId] = useState(tabs[0].name);
-    const [isPageOpen, setIsPageOpen] = useState(false);
+    const [activeTabId, setActiveTabId] = useState<string>(tabs[0].name);
+    const [isPageOpen, setIsPageOpen] = useState<boolean>(false);
 
     const query = useQuery();
     const queryTabValue = query?.get('tab');
 
-    // Run initially and when query is updated to set active tab based on initial URL
-    // or while navigating browser history
     useEffect(() => {
-        // Only change active tab when provided tab in query is specified in globalConfig
```

Comment:
why the comment was removed?

Reply:
it was due to some changes but have added in next push